script.js

```diff
@@ -1,4 +1,122 @@
+/*
+- player 1 rolls two dice and choose the order of the dice
+- player 2 rolls two dice and choose the order of the dice
+- compare player 1 and player 2 number, state the winner
+- reset the game
+*/
+
+var gameState = "player1Roll";
```

Comment:
for game state its better to list all your game states as global variables at the top then update it based on the variable name in your code

example:
```
const PLAYER_1_ROLL = "player1Roll"
const PLAYER_2_ROLL = "player2Roll"
...

var gameState = PLAYER_1_ROLL;
```

then in line 40 and 87 for example, where you are checking or updating game state to player2Roll, you do `gameState = PLAYER_2_ROLL`

Reply:
benefit of this:
- only having to change the string value in one place, if ever you wanna update it
- less chance of typos leading to bugs, lets say in line 40 you type "player1roll" and line 87 you type "player1Roll"